### DISCUSSION TITLE
Please help with AlphineJS

### DISCUSSION THREAD

**Discussion by monointeractive** [livewire/livewire#6947] [4. General]
Why doesn't the alphinejs code work in the blade file (which is not a livewire component)? I thought that by installing livewire I can execute aplhinejs code?

 test.blade.php:
```
    <div class="ml-4" x-on:click="alert('Hello World!')">
 
    </div>
```

**Comment by arthurshlain** [comment]
Did you have `x-data` directive?

```html
<div x-data class="ml-4" x-on:click="alert('Hello World!')">
 
</div>
```

https://alpinejs.dev/directives/data#data-less-components

**Reply by monointeractive** [reply]
Great, it works. Thank you very much for help :)
